packages/web_drivers/README.md

```diff
@@ -1,23 +1,19 @@
-# To install and start Chrome Driver.
+**To install and start Chrome Driver.**
```

Comment:
Up to you, but why not keep it as normal text?

Reply:
I like different shades. It makes easier for me concentrate while reading.